### PR TITLE
Update populate.md

### PR DESCRIPTION
**Pull request by euj1n0ng**

### CODE REVIEW

File: docs/typescript/populate.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface Parent {
   name?: string
 }
 const ParentModel = model<Parent>('Parent', new Schema({
-  child: { type: 'ObjectId', ref: 'Child' },
+  child: { type: Types.ObjectId, ref: 'Child' },
```
Comment (Uzlopak) [inline docs/typescript/populate.md:14]: Actually I think it should be Schema.Types.ObjectId

Comment (euj1n0ng) [reply docs/typescript/populate.md:14]: That's because we've already imported `Types` from `mongoose`. :)

Comment (Uzlopak) [reply docs/typescript/populate.md:14]: Neh, it is a common misconception. In the interface you use Types.ObjectId but when you instantiate the Schema you use Schema.Types.ObjectId.

Comment (euj1n0ng) [reply docs/typescript/populate.md:14]: I've already allowed edits by maintainers. :)

Comment (Uzlopak) [reply docs/typescript/populate.md:14]:
```suggestion
  child: { type: Schema.Types.ObjectId, ref: 'Child' },
```

File: docs/typescript/populate.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ interface PopulatedParent {
   child: Child | null;
 }
 const ParentModel = model<Parent>('Parent', new Schema({
-  child: { type: 'ObjectId', ref: 'Child' },
+  child: { type: Types.ObjectId, ref: 'Child' },
```
Comment (Uzlopak) [inline docs/typescript/populate.md:48]:
```suggestion
  child: { type: Schema.Types.ObjectId, ref: 'Child' },
```

**Pull request merged**